editor.planx.uk/src/pages/OfflinePage.tsx

```diff
@@ -0,0 +1,11 @@
+import Typography from "@mui/material/Typography";
+import StatusPage from "pages/Preview/StatusPage";
+import React from "react";
+
+export const OfflinePage: React.FC = () => (
+  <StatusPage bannerHeading="Offline">
+    <Typography variant="body2">
+      This service is not currently available. Please check back later.
```

Comment:
Here's a maybe curveball question / suggestion (sorry!): 
>This service is not currently available to new applicants. Please check back later. 
>
>If you're resuming an application you previously started, please use the link sent to you via email.

Basically, what's our plan for handling active sessions for an "offline" service? Can we disable the `/published` route when no query params are present, but _enable_ it for folks coming with query params `email` & `sessionId` ? Otherwise I fear we get into much murkier questions around extending session expiry periods if offline, etc.

Reply:
Also I can imagine a future enhancement to the editor form could be simple rich text input to control what your "Offline" page says (eg add a specific date if they choose - like after winter holidays). But default copy would be this?

Reply:
Thanks for copy suggestion, updated ✅ 

No need to apologise - it's good to raise and be explicit here about the behaviour. I agree that this is actually the smartest way forward, and best for users. I've made change which implements this, but as it needs to be at a "higher level" than Save and Return, I've opted to use a layout which wraps this view.